### DISCUSSION TITLE
제페토 스튜디오의 package 파일을 등록시 월드 빌드 실패라고 뜹니다

### DISCUSSION THREAD

**Discussion by kiuworld** [naverz/zepeto-studio-kor#1194] [4. General]
유니티를 시작한지 얼마 되지 않아서 잘모르는데 
zepeto package 파일의 용량은 약 970메가 이며 
유니티의 패키지파일 , 스크립트를 전부 업데이트 하여도 계속하여 오류가 발생합니다 도와주세요 ㅠㅠ 
혹시나 하여 dl_zepeto_evangelist@naverz-corp.com 메일로 패키지 파일을 첨부합니다...
![안됨](https://user-images.githubusercontent.com/111574894/185617349-c0a11ca8-06a1-4ee1-ab50-ac9c289c1ab7.PNG)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 보내주신 메일 확인하였습니다.
내부에서 테스트 후 답변 드리겠습니다.
감사합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 크리에이터님 메일로 답변 드렸습니다.
감사합니다.
